### ISSUE TITLE
Syntax to add psx certificate from policies.json

### ISSUE THREAD

**Issue by raviteja8k** [mozilla/policy-templates#940]
I am trying to install a password protected pfx certificate into firefox using policies.json. What is the syntax to use password?

**Comment by mkaply** [comment]
We have no way to specify the password in this case.

I would recommend installing it at the OS level and setting Firefox to read OS certificates using ImportEnterpriseRoots

**Comment by raviteja8k** [comment]
I tried the following, but it didn't work.

{
  "policies": {
    "Certificates": {
      "ImportEnterpriseRoots": true
    }
  }
}

**Comment by mkaply** [comment]
when you go to about:policies, do you see the policy being set there?

**Comment by raviteja8k** [comment]
Yes the policy is being set. But under errors, it shows as unable to install the certificate and I believe it is because my cert needs a password and I don't have an option to give password from policies.json.

**Comment by raviteja8k** [comment]
My json...
{
"policies":{
"Certificates" : {
"ImportEnterpriseRoots": true,
"Install": ["add_client.pfx"]
}
}
}

**Comment by mkaply** [comment]
You would have to create a new cert without a password.
